### DISCUSSION TITLE
Where can I find the results of the cachedResolution?

### DISCUSSION THREAD

**Discussion by jeanmarc** [sbt/sbt#7176] [Q&A] [answered]
See also SO: https://stackoverflow.com/questions/74813764/where-is-the-sbt-cached-resolution-stored (hasn't received an answer yet, posted 2 months ago):

According to [1], project reloads should be sped up by adding

`ThisBuild / updateOptions := updateOptions.value.withCachedResolution(true)` 
To the build.sbt.

I've done that, and now I want to inspect these cached results (basically to confirm that the setting is active and actually works).

I have looked at the mentioned folder ($HOME/.sbt/1.0/dependency), but there is nothing at that location.

If I start sbt shell and do print dependencyCacheDirectory, it lists that folder for each of the sbt modules (I have a multi-module project, with about 200 modules :) ):

```
> print dependencyCacheDirectory
...
dependencyCacheDirectory
    /Users/jml/.sbt/1.0/dependency
```

Using sbt version 1.8.3 (project sbt version 1.7.2), IntellIJ 2022.3 with BSP.

[1] - https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html#Cached+Resolution

**Comment by jtjeferreira** [comment, marked as answer]
I think that feature only applies to ivy and not to coursier...

**Reply by jeanmarc** [reply]
I see. I was under the impression that the dependencyCacheDirectory would keep track of dependencies between sub-modules (speeding up the overall dependency analysis by being able to re-use repetitive `dependsOn` (sub)dags). Re-reading https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html with only libraryDependencies in mind makes it clear that that is not what this cached resolution feature was aimed at.
